core/distributed/preconditioner/schwarz.cpp

```diff
@@ -0,0 +1,120 @@
+/*******************************<GINKGO LICENSE>******************************
+Copyright (c) 2017-2022, the Ginkgo authors
```

Comment:
I guess the year needs to updated to 2023
```suggestion
Copyright (c) 2017-2023, the Ginkgo authors
```

Reply:
I guess you can just run a sed command to fix that in the other files ...